libraries/core-react/src/Divider/index.ts

```diff
@@ -1 +1 @@
-export { Divider, DividerProps } from './Divider'
+export * from './Divider'
```

Comment:
By re-exporting `*` we don’t get error messages about the types not being exported.

Reply:
Just did kinda the same for Banner